napari/components/_viewer/view/main.py

```diff
@@ -62,8 +62,18 @@ def __init__(self, viewer):
                 'standard': QCursor()
             }
 
-    def set_cursor(self, cursor):
-        self.canvas.native.setCursor(self._cursors[cursor])
+    def set_cursor(self, cursor, size=None):
+        if cursor == 'square':
+            if size is None:
+                size = 10
```

Comment:
why don't you just make the default size 10?

Reply:
done